pandas/plotting/_converter.py

```diff
@@ -178,6 +180,16 @@ class DatetimeConverter(dates.DateConverter):
 
     @staticmethod
     def convert(values, unit, axis):
+        # values might be a 1-d array, or a list-like of arrays.
+        if is_list_like(values) and is_list_like(values[0]):
```

Comment:
A (minor) problem with `all(is_list_like(v) for v in values)` is that this returns True for an empty list (the `values=[]` case). 

It's only minor in that it still works but the final return result will become an empty list instead of an empty float array. Which might be a problem. 

(I am not sure to what extent a Converter is supposed to return an array in matplotlib code)